tactics/Goalie.py

```diff
@@ -88,7 +88,10 @@ def add_kub(self,kub):
 		self.kub = kub
 
 	def peace_out(self):
+<<<<<<< HEAD
 		# state = shared.get('state')
+=======
+>>>>>>> fsm
```

Comment:
You have to remove this.

Reply:
In tactics/Goalie.py, This has to be removed. This is due to the merge conflict.